.gitlab-ci.yml

```diff
@@ -19,10 +20,34 @@ driver_build:
     - mkdir artifacts
     - xcopy /e/s build-out\${CI_JOB_ID}\*.* artifacts
     - remove-item -recurse -force build-out\${CI_JOB_ID}
-    - aws s3 cp artifacts/ s3://dd-windowsfilter/builds/tracer/${CI_COMMIT_SHA} --recursive --region us-east-1 --exclude "*" --include "*.zip" --include "*.msi"
     - get-childitem build-out
     - get-childitem artifacts
   artifacts:
     expire_in: 2 weeks
     paths:
-    - artifacts
\ No newline at end of file
+    - artifacts
+
+publish:
+  only:
+    - master
+    - main
+  stage: publish
+  tags: ["runner:windows-docker", "windowsversion:1809"]
+  dependencies: 
+    - build
+  script:
+    - $result =  aws sts assume-role --role-arn "arn:aws:iam::486234852809:role/ci-datadog-windows-filter" --role-session-name AWSCLI-Session
+    - $resultjson = $result | convertfrom-json
+    - $credentials = $($resultjson.Credentials)
+    - $Env:AWS_ACCESS_KEY_ID="$($credentials.AccessKeyId)"
+    - $Env:AWS_SECRET_ACCESS_KEY="$($credentials.SecretAccessKey)"
+    - $Env:AWS_SESSION_TOKEN="$($credentials.SessionToken)"
```

Comment:
Are you sure this is the correct way to set these 🤔 Doesn't this expose them in the logs?

Reply:
> correct way

100% sure this is the best way, no, clearly. That said:
- it's used ~often in dd already (cf [codesearch](https://cs.github.com/?scopeName=DataDog&scope=org%3ADataDog&q=aws+sts+assume-role))
- it does not print out the keys (cf [log](https://gitlab.ddbuild.io/DataDog/dd-trace-dotnet/-/jobs/111710362#L34))